### ISSUE TITLE
Input wheel no longer visible  

### ISSUE THREAD

**Issue by carlosgamezvillegas** [kovalev-sergey/homebridge-sony-audio#42]
**Describe The Bug:**

The input selector is no longer available
**To Reproduce:**

Just open the accessory. The only thing visible is the power button
**Expected behavior:**

See the input selector to change inputs 
**Logs:**

```
Show the Homebridge logs here, remove any sensitive information.
```

**Plugin Config:**

```json
Show your Homebridge config.json here, remove any sensitive information.
```

**Screenshots:**

![image](https://github.com/kovalev-sergey/homebridge-sony-audio/assets/67111609/b616da8a-ad0c-42ff-9fd0-a467a2d8e5e0)

**Environment:**

* **Plugin Version**: 1.2.0
* **Homebridge Version**:  1.7
* **Node.js Version**:  16.20
* **NPM Version**:  n/a
* **Operating System**:  Raspbian

**Comment by REBELinBLUE** [comment]
Did you manage to get this fixed?

**Comment by carlosgamezvillegas** [comment]
I don’t have an issue anymore running iOS 18.

**Comment by REBELinBLUE** [comment]
Interesting, I absolutely do still have the issue on the one device updated to iOS 18, but I haven't updated the HomePods so maybe when I do that it'll be fixed

**Comment by REBELinBLUE** [comment]
Sadly it's still not back for me with iOS 18.1 on all devices

**Comment by carlosgamezvillegas** [comment]
I don't see the wheel anymore :(

**Comment by REBELinBLUE** [comment]
Yeah, I setup the plugin again as a child bridge and the wheel reappeared but after a day or so vanished again

**Comment by kovalev-sergey** [comment]
Root cause found and fixed in #55: the `InputSource` services were created in an unguarded promise chain (`settings → getInputs → buildInputs`). If the device was asleep or did not answer while the accessory was being created — which typically happens right after a Homebridge/child bridge restart — the chain rejected silently, no input was ever created and nothing retried it. HomeKit then showed the Television with the power button only until the next restart, which matches the "it came back after re-adding it, then vanished again a day later" reports here.

What changed:
* the inputs are now built by a guarded routine that logs the failure and retries every 5 seconds;
* the accessory is published only after its inputs exist, so HomeKit never sees a Television without input sources;
* additionally `ActiveIdentifier` is now updated for the first input as well (identifier `0` was treated as "not found"), and a persisted custom input name is no longer reset to the default (#54).

All of this is covered by regression tests and ships in 2.0.1.
